cassandra/io/twistedreactor.py

```diff
@@ -222,7 +265,7 @@ def close(self):
             self.is_closed = True
 
         log.debug("Closing connection (%s) to %s", id(self), self.host)
-        self.connector.disconnect()
+        reactor.callFromThread(self.connector.disconnect)
```

Comment:
What made this `callFromThread` necessary? And maybe this should use `twisted.internet.blockingCallFromThread` -- I believe that the bare `disconnect` call would block, but I don't think this does.

Reply:
This is independent from the ssl addition. `disconnect` should be called from twisted event loop thread since it's not threasafe. We do the same in [asyncore](https://github.com/datastax/python-driver/blob/de86392bd68b479f52510f631cca08c5c2302f4c/cassandra/io/asyncorereactor.py#L373). I don't think there's anything wrong with not waiting, we just schedule it and it will happen.

Reply:
Got it :+1: sgtm